week-1/Homework/mandatory/1-syntax-errors.js

```diff
@@ -1,17 +1,26 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
     return a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+<<<<<<< Updated upstream
+return "Hello, my name is " + name + " and I am " + age + " years old";
+=======
+return (`Hello, my name is ${name} and I am ${age} years old`);
+>>>>>>> Stashed changes
+}
 
 function getRemainder(a, b) {
-    remainder = a %% b;
+    remainder = a % b;
```

Comment:
I think there's something missing in front of `remainder` 🙂

Reply:
thank you for pointing it out, I've amended the mistake